5.0/en/0x10-V1-Encoding-Sanitization.md

```diff
@@ -61,15 +61,19 @@ Where it is not possible to do this, sanitization will be necessary where potent
 
 ## V1.4 Memory, String, and Unmanaged Code
 
-The following requirements cover risks around unsafe memory use and will generally only apply when the application uses a systems language or unmanaged code.
+The following requirements cover risks around unsafe memory use and will generally only apply when the application uses a system's language or unmanaged code.
 
 In some cases, it may be possible to achieve this by setting compiler flags which enable buffer overflow protections and warnings, including stack randomization, data execution prevention, and will break the build if an unsafe pointer, memory, format string, integer, or string operations are found.
 
 | # | Description | Level | #v5.0.be |
 | :---: | :--- | :---: | :---: |
-| **1.4.1** | Verify that the application uses memory-safe string, safer memory copy and pointer arithmetic to detect or prevent stack, buffer, or heap overflows. | 2 | v5.0.be-5.4.1 |
+| **1.4.1** | Verify that the application uses memory-safe string, safer memory copy, and pointer arithmetic to detect or prevent stack, buffer, or heap overflows. | 2 | v5.0.be-5.4.1 |
 | **1.4.2** | Verify that sign, range, and input validation techniques are used to prevent integer overflows. | 2 | v5.0.be-5.4.3 |
-| **1.4.3** | Verify that dynamically allocated memory and resources are released, and that references or pointers to freed memory are removed or set to null to prevent dangling pointers and use-after-free vulnerabilities. | 2 | v5.0.be-5.4.4 |
+| **1.4.3** | Verify that dynamically allocated memory and resources are released, and that references or pointers to freed memory are removed or set to null to prevent dangling pointers and use-after-free vulnerabilities. | 3 | v5.0.be-5.4.4 |
```

Comment:
Please remove these proposed requirements to an issue, following [the guidance mentioned here](https://github.com/OWASP/ASVS/pull/3029#issuecomment-2823020749).